tests/parsing/GlobbingTest.py

```diff
@@ -211,10 +217,11 @@ def test_multiple_patterns(self):
         non_matches = ['aXbX', 'aXcX']
         self._test_fnmatch(pattern, matches, non_matches)
 
+    def test_empty_globs(self):
         pattern = []
         matches = ['anything', 'anything_else']
         non_matches = []
```

Comment:
Shouldn't this technically return false as well? With no pattern to match, there will never be a match, even if the list to be checked is empty.